CONTRIBUTING.md

```diff
@@ -140,6 +140,8 @@ You can use any editor you like, but if you use [Visual Studio Code](https://cod
 
 To enable correct validation and to fix lint/style errors on save, add this to your VSCode `settings.json`:
 
+<!-- eslint-disable -->
```

Comment:
Maybe we can just add colons around the JSON block, to make it valid JSON?

Reply:
Done